account_invoice_refund_reason/models/account_move_refund_reason.py

```diff
@@ -5,7 +5,7 @@
 
 
 class AccountInvoiceRefundReason(models.Model):
-    _name = "account.invoice.refund.reason"
+    _name = "account.move.refund.reason"
```

Comment:
I don't think you should change the model name.
Otherwise, you will need migration script for this. WDYT?